source/_integrations/sensor.avri.markdown

```diff
@@ -0,0 +1,52 @@
+---
+title: Avri Waste Sensor
+description: Instructions on how to set up Avri Waste sensor within Home Assistant.
+logo: avri.jpeg
+ha_category: 
+  - Sensor
+ha_iot_class: "Cloud Polling"
+ha_release: '0.106'
+ha_codeowners:
+  - '@timvancann'
+---
+
+The `Avri Waste` platform allows you to track the next scheduled waste pickup and what type of waste from [Avri](https://www.avri.nl/). 
+
+## Configuration
+
+To enable this sensor, add the following lines to your `configuration.yaml`:
+
+```yaml
+# Example configuration.yaml entry
+sensor:
+  - platform: avri
+    postcode: YOUR_POST_CODE
+    house_number: YOUR_HOUSE_NUMBER
+```
+
+{% configuration %}
+postcode:
```

Comment:
This is dutch...

I would expect it to be `postal_code` or `zip_code`

Reply:
Thinking about it now I would prefer `zip_code` since it is used in [rova](https://github.com/home-assistant/home-assistant/blob/10e698e0b217e7553989caea708c02e38fb19fb4/homeassistant/components/rova/sensor.py#L21), [iqvia](https://github.com/home-assistant/home-assistant/blob/f75bda75512e36faaebddcd314149553d99d9f6f/homeassistant/components/iqvia/const.py#L4) and [flunearyou](https://github.com/home-assistant/home-assistant/blob/f75bda75512e36faaebddcd314149553d99d9f6f/homeassistant/components/flunearyou/sensor.py#L30).

Reply:
Made it `zip_code`, modified the code accordingly as well.